### DISCUSSION TITLE
Which version of Phalcon can I use now?

### DISCUSSION THREAD

**Discussion by dlusignan** [phalcon/phalcon#223] [Q&A]
Hi,

I'm using version 3 and I want to upgrade. Which version is the stable one? 4-5 or 6?

Thank you.

**Comment by Jeckerson** [comment]
Hello, you can use v4.1 with PHP7.4. After several months - v5 or v6 with PHP >= 7.4

**Reply by dlusignan** [reply]
So, it's version 4 only for now? Both 5 and 6 are in development?

**Reply by Arhell** [reply]
Yes
https://blog.phalcon.io/post/phalcon-5-0-0alpha7-released
https://blog.phalcon.io/post/phalcon-roadmap
